specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/appcomplianceautomation.json

```diff
@@ -0,0 +1,1374 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceGroup": {
+          "description": "The resource group name of this resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceType": {
+          "description": "The resource type of this resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceId": {
+          "description": "The resource Id.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DownloadResponse": {
+      "description": "Object that includes all the possible response for the download operation.",
+      "type": "object",
+      "properties": {
+        "resourceList": {
+          "description": "List of the reports",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ResourceItem"
+          },
+          "x-ms-identifiers": []
+        },
+        "complianceReport": {
+          "description": "List of the compliance result",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ComplianceReportItem"
+          },
+          "x-ms-identifiers": []
+        },
+        "compliancePdfReport": {
+          "description": "compliance pdf report",
+          "type": "object",
+          "readOnly": true,
+          "properties": {
+            "sasUri": {
+              "description": "uri of compliance pdf report",
+              "type": "string",
+              "readOnly": true,
+              "x-ms-secret": true
+            }
+          }
+        },
+        "complianceDetailedPdfReport": {
+          "description": "compliance detailed pdf report",
+          "type": "object",
+          "readOnly": true,
+          "properties": {
+            "sasUri": {
+              "description": "uri of compliance detailed pdf report",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "ComplianceReportItem": {
+      "type": "object",
+      "description": "Object that includes all the content for single compliance result.",
+      "properties": {
+        "categoryName": {
+          "type": "string",
+          "description": "The category name.",
+          "readOnly": true
+        },
+        "controlId": {
+          "type": "string",
+          "description": "The control Id.",
+          "readOnly": true
+        },
+        "controlName": {
+          "type": "string",
+          "description": "The control name.",
+          "readOnly": true
+        },
+        "controlType": {
+          "type": "string",
+          "description": "The control type.",
+          "$ref": "#/definitions/ControlType",
+          "readOnly": true
+        },
+        "complianceState": {
+          "type": "string",
+          "description": "The compliance result's status.",
```

Comment:
>description [](http://example.com/codeflow?start=11&length=11)

Is there an enum of known values? Or is it freeform?

Reply:
Yes, we have changed complianceState to enum, the available values can be "Healthy" and "Unhealthy".
Thanks Tim~